### ISSUE TITLE
can't download white icon

### ISSUE THREAD

**Issue by rahmanshaber** [material-components/material-components#30]
i tried to download white svg in 18d | 24d .
but it downloaded black one.

**Comment by ajsecord** [comment]
I can reproduce this bug. Repro steps for tracking:

1. Go to https://material.io/tools/icons/?icon=3d_rotation&style=baseline
1. Expand the left nav.
1. Click on the white circle to choose a white version.
1. Download the SVG.
1. Open in Sketch or other editor.

Desired outcome: the icon is white.
Observed outcome: the icon is black.

I'll file an internal issue and keep this open for now.

**Comment by ajsecord** [comment]
Internal issue 79758093 for tracking.

**Comment by ajsecord** [comment]
Duped to internal issue 79614663 for tracking.

**Comment by rahmanshaber** [comment]
yes i tried everything .
i opend it with gimp(editor)
same thing not, downloding white icon .black black.

i downloaded white and black icon before redesign the website.
i think it is website issue.

**Comment by rahmanshaber** [comment]
so how much time it will take to fix it??
my app release is on hold for this.

**Comment by willlarche** [comment]
You should not wait on this.

**Comment by rahmanshaber** [comment]
did not expected that

**Comment by ajsecord** [comment]
Hi, I'm sorry this is still blocking you. Here's a workaround while we fix the internal issue:

1. Download the icon in black.
1. Go to http://vectorpaint.yaks.co.nz/
1. File > Open SVG, then select the file you downloaded.
1. With the arrow tool, select the icon.
1. Fill it with white by clicking on the white "Fill" circle on the right.
1. File > Download to download the edited SVG file.

Hope this helps!

**Comment by rahmanshaber** [comment]
i tried your way but it needs on-line connection.
but i did this in inkscape.

also i noticed that the black icon color is shown in the website is not the same as the download one.(i am  still not able to download the white one)

**Comment by DaanielSR** [comment]
Hi , i cant import in Android Studio the SVG downloaded from that page , the page of material design still does not work.

**Comment by levirgon** [comment]
still getting the same issue.

**Comment by MonstraG** [comment]
Can confirm. Android studio cannot import the file after changing color to white using the site.

**Comment by cketcham** [comment]
You should be able to download the svg, import into android studio and then change `android:fillColor` for each path in the xml of the vector drawable that was created.

**Comment by MonstraG** [comment]
All those icons actually already available in "Clip art" asset type section in Android studio.

**Comment by yDkay** [comment]
Still now working, i'm downloading the white icon and getting the black one.

**Comment by mfsi-prateeksharma** [comment]
A quick solution is removing the tint from the drawable XML. 
For eg. 
```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
        android:width="24dp"
        android:height="24dp"
        android:viewportWidth="24.0"
        android:viewportHeight="24.0"
        android:tint="?attr/colorControlNormal"> // Remove this
    <path
        android:fillColor="@android:color/white"
        android:pathData="M20,11H7.83l5.59,-5.59L12,4l-8,8 8,8 1.41,-1.41L7.83,13H20v-2z"/>
</vector>
```

**Comment by asfarley** [comment]
Still not working, 2 years after being reported

**Comment by pierresis** [comment]
Still not working...

**Comment by DaanielSR** [comment]
we know that after 2 years this still not working , seems nobody will fix
it soon , so we need to look for in other side

**Comment by leticiarossi** [comment]
This issue has been fixed
